### PR TITLE
Correct docs

### PR DESCRIPTION
**Pull request by zhangrui-wolf**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

The document was corrected and some new additions to the English document were translated.

## Modification

All Chinese documents.

## BC-breaking (Optional)

No.

## Use cases (Optional)

No.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects, like MMDet or MMDet3D.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

**Comment by Junjun2016** [comment]
Hi @zhangrui-wolf 
Thanks for your nice PR!
Please sign the CLA!
And we will review it ASAP!

**Comment by Junjun2016** [comment]
Hi @zhangrui-wolf 
Please use pre-commit to check and fix the lint error.

**Comment by MengzhangLI** [comment]
Hi, thanks for your effort!

I just scan your PR, it does not fix typo, it tries to make the translation more faithful, expressive and elegant.

We would take it PR for consideration and thanks again for your enthusiasm!

Best,

File: docs_zh-CN/dataset_prepare.md [hunk 1]
```diff
@@ -73,12 +73,9 @@ mmsegmentation
 
 ### Cityscapes
 
-注册成功后，数据集可以在 [这里](https://www.cityscapes-dataset.com/downloads/) 下载。
+注册成功后可以在 [这里](https://www.cityscapes-dataset.com/downloads/) 找到Cityscapes相关数据。
```
Comment (MengzhangLI) [inline docs_zh-CN/dataset_prepare.md:76]: OK.

File: docs_zh-CN/dataset_prepare.md [hunk 2]
```diff
@@ -73,12 +73,9 @@ mmsegmentation
...
 
-通常情况下，`**labelTrainIds.png` 被用来训练 cityscapes。
-基于 [cityscapesscripts](https://github.com/mcordts/cityscapesScripts),
-我们提供了一个 [脚本](https://github.com/open-mmlab/mmsegmentation/blob/master/tools/convert_datasets/cityscapes.py),
-去生成 `**labelTrainIds.png`。
+按照惯例，`**labelTrainIds.png` 被用来训练。我们提供了一个基于 [cityscapesscripts](https://github.com/mcordts/cityscapesScripts)的 [脚本](https://github.com/open-mmlab/mmsegmentation/blob/master/tools/convert_datasets/cityscapes.py)来生成 `**labelTrainIds.png`。
```
Comment (MengzhangLI) [inline docs_zh-CN/dataset_prepare.md:78]: Not good, too many changes.

File: docs_zh-CN/dataset_prepare.md [hunk 3]
```diff
@@ -87,41 +84,36 @@ python tools/convert_datasets/cityscapes.py data/cityscapes --nproc 8
 
 ### Pascal VOC
 
-Pascal VOC 2012 可以在 [这里](http://host.robots.ox.ac.uk/pascal/VOC/voc2012/VOCtrainval_11-May-2012.tar) 下载。
-此外，许多最近在 Pascal VOC 数据集上的工作都会利用增广的数据，它们可以在 [这里](http://www.eecs.berkeley.edu/Research/Projects/CS/vision/grouping/semantic_contours/benchmark.tgz) 找到。
+Pascal VOC 2012 可以在 [这里](http://host.robots.ox.ac.uk/pascal/VOC/voc2012/VOCtrainval_11-May-2012.tar) 下载。此外，最近在 Pascal VOC 数据集上的大多数工作都会利用额外的增广数据，这些数据可以在 [这里](http://www.eecs.berkeley.edu/Research/Projects/CS/vision/grouping/semantic_contours/benchmark.tgz) 找到。
```
Comment (MengzhangLI) [inline docs_zh-CN/dataset_prepare.md:87]: Not good, no need to change original setting.

File: docs_zh-CN/dataset_prepare.md [hunk 4]
```diff
@@ -87,41 +84,36 @@ python tools/convert_datasets/cityscapes.py data/cityscapes --nproc 8
...
 
-如果您想使用增广后的 VOC 数据集，请运行下面的命令来将数据增广的标注转成正确的格式。
+如果您想使用增广后的 VOC 数据集，请运行以下命令，将数据增广的标注转成正确的格式。
```
Comment (MengzhangLI) [inline docs_zh-CN/dataset_prepare.md:89]: OK.

File: docs_zh-CN/dataset_prepare.md [hunk 5]
```diff
@@ -87,41 +84,36 @@ python tools/convert_datasets/cityscapes.py data/cityscapes --nproc 8
...
-注册成功后，您还可以在 [这里](http://host.robots.ox.ac.uk:8080/eval/downloads/VOC2010test.tar) 下载验证集。
+Pascal Context 的训练集和验证集可以从 [这里](http://host.robots.ox.ac.uk/pascal/VOC/voc2010/VOCtrainval_03-May-2010.tar) 下载。注册成功后，您还可以从 [这里](http://host.robots.ox.ac.uk:8080/eval/downloads/VOC2010test.tar) 下载验测试集。
 
-为了从原始数据集里切分训练集和验证集， 您可以在 [这里](https://codalabuser.blob.core.windows.net/public/trainval_merged.json)
-下载 trainval_merged.json。
+为了从原始数据集中切分出训练集和验证集， 您可以从 [这里](https://codalabuser.blob.core.windows.net/public/trainval_merged.json)下载 trainval_merged.json。
 
-如果您想使用 Pascal Context 数据集，
-请安装 [细节](https://github.com/zhanghang1989/detail-api) 然后再运行如下命令来把标注转换成正确的格式。
+如果您想使用 Pascal Context 数据集，请安装 [Detail API](https://github.com/zhanghang1989/detail-api) ，然后运行以下命令来把标注转换成正确的格式。
```
Comment (MengzhangLI) [inline docs_zh-CN/dataset_prepare.md:108]: Not good, no need to change original setting.

File: docs_zh-CN/dataset_prepare.md [hunk 6]
```diff
@@ -87,41 +84,36 @@ python tools/convert_datasets/cityscapes.py data/cityscapes --nproc 8
...
 
 ### Pascal Context
 
-Pascal Context 的训练集和验证集可以在 [这里](http://host.robots.ox.ac.uk/pascal/VOC/voc2010/VOCtrainval_03-May-2010.tar) 下载。
-注册成功后，您还可以在 [这里](http://host.robots.ox.ac.uk:8080/eval/downloads/VOC2010test.tar) 下载验证集。
+Pascal Context 的训练集和验证集可以从 [这里](http://host.robots.ox.ac.uk/pascal/VOC/voc2010/VOCtrainval_03-May-2010.tar) 下载。注册成功后，您还可以从 [这里](http://host.robots.ox.ac.uk:8080/eval/downloads/VOC2010test.tar) 下载验测试集。
 
-为了从原始数据集里切分训练集和验证集， 您可以在 [这里](https://codalabuser.blob.core.windows.net/public/trainval_merged.json)
-下载 trainval_merged.json。
+为了从原始数据集中切分出训练集和验证集， 您可以从 [这里](https://codalabuser.blob.core.windows.net/public/trainval_merged.json)下载 trainval_merged.json。
```
Comment (MengzhangLI) [inline docs_zh-CN/dataset_prepare.md:106]: Not good, no need to change original setting.

File: docs_zh-CN/dataset_prepare.md [hunk 7]
```diff
@@ -87,41 +84,36 @@ python tools/convert_datasets/cityscapes.py data/cityscapes --nproc 8
...
 
-ADE20K 的训练集和验证集可以在 [这里](http://data.csail.mit.edu/places/ADEchallenge/ADEChallengeData2016.zip) 下载。
-您还可以在 [这里](http://data.csail.mit.edu/places/ADEchallenge/release_test.zip) 下载验证集。
+ADE20K 的训练集和验证集可以从 [这里](http://data.csail.mit.edu/places/ADEchallenge/ADEChallengeData2016.zip) 下载。您还可以在 [这里](http://data.csail.mit.edu/places/ADEchallenge/release_test.zip) 下载测试集。
 
 ### Pascal Context
 
-Pascal Context 的训练集和验证集可以在 [这里](http://host.robots.ox.ac.uk/pascal/VOC/voc2010/VOCtrainval_03-May-2010.tar) 下载。
-注册成功后，您还可以在 [这里](http://host.robots.ox.ac.uk:8080/eval/downloads/VOC2010test.tar) 下载验证集。
+Pascal Context 的训练集和验证集可以从 [这里](http://host.robots.ox.ac.uk/pascal/VOC/voc2010/VOCtrainval_03-May-2010.tar) 下载。注册成功后，您还可以从 [这里](http://host.robots.ox.ac.uk:8080/eval/downloads/VOC2010test.tar) 下载验测试集。
```
Comment (MengzhangLI) [inline docs_zh-CN/dataset_prepare.md:104]: Not good, no need to change original setting.

File: docs_zh-CN/dataset_prepare.md [hunk 8]
```diff
@@ -87,41 +84,36 @@ python tools/convert_datasets/cityscapes.py data/cityscapes --nproc 8
...
 ```
 
-关于如何拼接数据集 (concatenate) 并一起训练它们，更多细节请参考 [拼接连接 数据集](https://github.com/open-mmlab/mmsegmentation/blob/master/docs/tutorials/new_dataset.md#concatenate-dataset) 。
+请参考 [拼接数据集](https://github.com/open-mmlab/mmsegmentation/blob/master/docs/tutorials/new_dataset.md#concatenate-dataset) ，了解如何将它们拼接起来并一起用于训练。
 
 ### ADE20K
 
-ADE20K 的训练集和验证集可以在 [这里](http://data.csail.mit.edu/places/ADEchallenge/ADEChallengeData2016.zip) 下载。
-您还可以在 [这里](http://data.csail.mit.edu/places/ADEchallenge/release_test.zip) 下载验证集。
+ADE20K 的训练集和验证集可以从 [这里](http://data.csail.mit.edu/places/ADEchallenge/ADEChallengeData2016.zip) 下载。您还可以在 [这里](http://data.csail.mit.edu/places/ADEchallenge/release_test.zip) 下载测试集。
```
Comment (MengzhangLI) [inline docs_zh-CN/dataset_prepare.md:100]: Not good, no need to change original setting.

File: docs_zh-CN/dataset_prepare.md [hunk 9]
```diff
@@ -87,41 +84,36 @@ python tools/convert_datasets/cityscapes.py data/cityscapes --nproc 8
...
-如果您想使用增广后的 VOC 数据集，请运行下面的命令来将数据增广的标注转成正确的格式。
+如果您想使用增广后的 VOC 数据集，请运行以下命令，将数据增广的标注转成正确的格式。
 
 ```shell
 # --nproc 8 意味着有 8 个进程用来转换，它也可以被忽略。
 python tools/convert_datasets/voc_aug.py data/VOCdevkit data/VOCdevkit/VOCaug --nproc 8
 ```
 
-关于如何拼接数据集 (concatenate) 并一起训练它们，更多细节请参考 [拼接连接 数据集](https://github.com/open-mmlab/mmsegmentation/blob/master/docs/tutorials/new_dataset.md#concatenate-dataset) 。
+请参考 [拼接数据集](https://github.com/open-mmlab/mmsegmentation/blob/master/docs/tutorials/new_dataset.md#concatenate-dataset) ，了解如何将它们拼接起来并一起用于训练。
```
Comment (MengzhangLI) [inline docs_zh-CN/dataset_prepare.md:96]: OK.

File: docs_zh-CN/dataset_prepare.md [hunk 10]
```diff
@@ -87,41 +84,36 @@ python tools/convert_datasets/cityscapes.py data/cityscapes --nproc 8
...
+如果您想使用 Pascal Context 数据集，请安装 [Detail API](https://github.com/zhanghang1989/detail-api) ，然后运行以下命令来把标注转换成正确的格式。
 
 ```shell
 python tools/convert_datasets/pascal_context.py data/VOCdevkit data/VOCdevkit/VOC2010/trainval_merged.json
 ```
 
 ### CHASE DB1
 
-CHASE DB1 的训练集和验证集可以在 [这里](https://staffnet.kingston.ac.uk/~ku15565/CHASE_DB1/assets/CHASEDB1.zip) 下载。
+CHASE DB1 的训练集和验证集可以从 [这里](https://staffnet.kingston.ac.uk/~ku15565/CHASE_DB1/assets/CHASEDB1.zip) 下载。
```
Comment (MengzhangLI) [inline docs_zh-CN/dataset_prepare.md:116]: OK.

File: docs_zh-CN/dataset_prepare.md [hunk 11]
```diff
@@ -133,8 +125,7 @@ python tools/convert_datasets/chase_db1.py /path/to/CHASEDB1.zip
 
 ### DRIVE
 
-DRIVE 的训练集和验证集可以在 [这里](https://drive.grand-challenge.org/) 下载。
-在此之前，您需要注册一个账号，当前 '1st_manual' 并未被官方提供，因此需要您从其他地方获取。
+DRIVE 的训练集和验证集可以在 [这里](https://drive.grand-challenge.org/) 下载。在下载之前，您需要注册一个账号。官方当前没有提供 '1st_manual' ，因此需要您从其他地方获取。
```
Comment (MengzhangLI) [inline docs_zh-CN/dataset_prepare.md:128]: Not good, no need to change original setting.

File: docs_zh-CN/dataset_prepare.md [hunk 12]
```diff
@@ -146,7 +137,7 @@ python tools/convert_datasets/drive.py /path/to/training.zip /path/to/test.zip
...
+首先，下载 [healthy.zip](https://www5.cs.fau.de/fileadmin/research/datasets/fundus-images/healthy.zip)， [glaucoma.zip](https://www5.cs.fau.de/fileadmin/research/datasets/fundus-images/glaucoma.zip)， [diabetic_retinopathy.zip](https://www5.cs.fau.de/fileadmin/research/datasets/fundus-images/diabetic_retinopathy.zip)，[healthy_manualsegm.zip](https://www5.cs.fau.de/fileadmin/research/datasets/fundus-images/healthy_manualsegm.zip)， [glaucoma_manualsegm.zip](https://www5.cs.fau.de/fileadmin/research/datasets/fundus-images/glaucoma_manualsegm.zip) 以及 [diabetic_retinopathy_manualsegm.zip](https://www5.cs.fau.de/fileadmin/research/datasets/fundus-images/diabetic_retinopathy_manualsegm.zip)。
```
Comment (MengzhangLI) [inline docs_zh-CN/dataset_prepare.md:140]: OK.

File: docs_zh-CN/dataset_prepare.md [hunk 13]
```diff
@@ -158,7 +149,7 @@ python tools/convert_datasets/hrf.py /path/to/healthy.zip /path/to/healthy_manua
 
 ### STARE
 
-首先，下载 [stare-images.tar](http://cecas.clemson.edu/~ahoover/stare/probing/stare-images.tar), [labels-ah.tar](http://cecas.clemson.edu/~ahoover/stare/probing/labels-ah.tar) 和 [labels-vk.tar](http://cecas.clemson.edu/~ahoover/stare/probing/labels-vk.tar).
+首先，下载 [stare-images.tar](http://cecas.clemson.edu/~ahoover/stare/probing/stare-images.tar)， [labels-ah.tar](http://cecas.clemson.edu/~ahoover/stare/probing/labels-ah.tar) 和 [labels-vk.tar](http://cecas.clemson.edu/~ahoover/stare/probing/labels-vk.tar)。
```
Comment (MengzhangLI) [inline docs_zh-CN/dataset_prepare.md:152]: OK.

File: docs_zh-CN/get_started.md [hunk 14]
```diff
@@ -1,13 +1,13 @@
 ## 依赖
 
-- Linux or macOS (Windows下支持需要 mmcv-full，但运行时可能会有一些问题。)
+- Linux or macOS （Windows系统处于试验性支持阶段）
```
Comment (MengzhangLI) [inline docs_zh-CN/get_started.md:3]: Not good, just keep the same.

File: docs_zh-CN/get_started.md [hunk 15]
```diff
@@ -1,13 +1,13 @@
...
 - Python 3.6+
 - PyTorch 1.3+
-- CUDA 9.2+ (如果您基于源文件编译 PyTorch, CUDA 9.0也可以使用)
+- CUDA 9.2+ （如果您基于源文件编译 PyTorch，CUDA 9.0也是兼容的）
```
Comment (MengzhangLI) [inline docs_zh-CN/get_started.md:6]: OK.

File: docs_zh-CN/get_started.md [hunk 16]
```diff
@@ -1,13 +1,13 @@
...
 - GCC 5+
 - [MMCV](https://mmcv.readthedocs.io/en/latest/#installation)
 
-可编译的 MMSegmentation 和 MMCV 版本如下所示，请对照对应版本安装以避免安装问题。
+兼容的 MMSegmentation 和 MMCV 版本如下所示，请安装对应版本的 MMCV 以避免出现兼容问题。
```
Comment (MengzhangLI) [inline docs_zh-CN/get_started.md:10]: OK.

File: docs_zh-CN/get_started.md [hunk 17]
```diff
@@ -24,108 +24,103 @@
 | 0.7.0               | mmcv-full>=1.1.2, <1.2.0 |
 | 0.6.0               | mmcv-full>=1.1.2, <1.2.0 |
 
-注意: 如果您已经安装好 mmcv, 您首先需要运行 `pip uninstall mmcv`。如果 mmcv 和 mmcv-full 同时被安装，会报错 `ModuleNotFoundError`。
+注意：如果您安装了 MMCV，您首先需要运行 `pip uninstall mmcv` 进行卸载。如果 mmcv 和 mmcv-full 并存于环境，会出现 `ModuleNotFoundError` 错误。
```
Comment (MengzhangLI) [inline docs_zh-CN/get_started.md:27]: Not good.

File: docs_zh-CN/get_started.md [hunk 18]
```diff
@@ -24,108 +24,103 @@
...
 
-c. 按照 [官方教程](https://mmcv.readthedocs.io/en/latest/#installation) 安装 [MMCV](https://mmcv.readthedocs.io/en/latest/) 。
-`mmcv` 或 `mmcv-full` 和 MMSegmentation 均兼容，但对于 CCNet 和 PSANet，`mmcv-full` 里的 CUDA 运算是必须的。
+- **在 Linux 下安装 mmcv：**
 
-**在 Linux 下安装 mmcv:**
+   通过运行
 
-通过运行
+   ```shell
```
Comment (MengzhangLI) [inline docs_zh-CN/get_started.md:50]: Not good. Please do not change structure of doc in casual.

File: docs_zh-CN/get_started.md [hunk 19]
```diff
@@ -24,108 +24,103 @@
...
-
-可以安装好 mmcv-full (PyTorch 1.5 和 CUDA 10.1) 版本。
-其他 PyTorch 和 CUDA 版本的 MMCV 安装请参照[这里](https://mmcv.readthedocs.io/en/latest/#install-with-pip)
+   可以安装预编译的 mmcv-full （PyTorch 1.6 和 CUDA 10.1） 。其他 PyTorch 和 CUDA 版本的 MMCV 的安装请参照[这里](https://mmcv.readthedocs.io/en/latest/get_started/installation.html)。
 
-**在 Windows 下安装 mmcv (有风险):**
+- **在 Windows 下安装 mmcv （试验性）：**
 
-对于 Windows, MMCV 的安装需要本地 C++ 编译工具, 例如 cl.exe. 请添加编译工具至 %PATH%.
+   对于 Windows，MMCV 的安装需要本地 C++ 编译工具，例如 cl.exe。 请将编译器路径添加到 %PATH%。
```
Comment (MengzhangLI) [inline docs_zh-CN/get_started.md:58]: OK.

**Comment by zhangrui-wolf** [comment]
After reading your example, I think I have an idea of how to change it. I will do it again to make changes to the document. Maybe close this merge for now and I will submit it again as soon as possible afterwards.

**Comment by MengzhangLI** [comment]
Hi, @zhangrui-wolf !

Thanks again for your great work!

 We consider follow most change you made, please wait and I will change & add detailed comments today.

Best,

**Comment by zhangrui-wolf** [comment]
Okay. Thank you for your help. This is my first pr submission for openmmlab, and I shouldn't submit a lot of updates at once. I will read carefully and follow the community's specifications when I contribute again.

File: docs_zh-CN/dataset_prepare.md (hunk 3 above)
Comment (MengzhangLI) [inline docs_zh-CN/dataset_prepare.md:87]:
```suggestion
Pascal VOC 2012 可以在 [这里](http://host.robots.ox.ac.uk/pascal/VOC/voc2012/VOCtrainval_11-May-2012.tar) 下载。

此外，最近在 Pascal VOC 数据集上的大多数工作都会利用额外的增广数据，这些数据可以在 [这里](http://www.eecs.berkeley.edu/Research/Projects/CS/vision/grouping/semantic_contours/benchmark.tgz) 找到。
```

File: docs_zh-CN/get_started.md (hunk 14 above)
Comment (MengzhangLI) [inline docs_zh-CN/get_started.md:3]:
```suggestion
- Linux or macOS （Windows系统处于试验性支持阶段）
```
Linux or macOS (Windows下支持需要 mmcv-full，但运行时可能会有一些问题。)

**Comment by MengzhangLI** [comment]
Hi, @zhangrui-wolf, sorry for late reply!

Thanks again for detailed reading and updating our chinese docs. We would accept most parts of your changes if we do not leave out comments.

Please check out my comments and fix them. There are two main problems:
(1) Leave out the a space between Chinese and English words, especially those sentences you added in the bottom of `useful_tools.md`.
(2) Just keep the order number the same with the original version.

Best,

**Comment by zhangrui-wolf** [comment]
I've modified useful_tools.md. You can check it first. I basically kept the structure of the original Chinese and English documentation, changed some typos and modified the content of TorchServe by referring to mmcls. If it is OK, I will follow this style of documentation to modify the rest of the documentation.

**Comment by MengzhangLI** [comment]
Hi, @zhangrui-wolf sorry for late reply! Excellent works you've done, thanks again.

You could follow your change on the other documents. Blanks are only needed between CHINESE and ENGLISH characters, no need to have it near symbols such as `（` or `）`.

Besides, if time available, could you help us check the correct terminology in Chinese? Here is the [link](https://github.com/open-mmlab/mmcv/blob/master/TERMINOLOGY.md).

image

Or you can just pulled your request as soon as possible and I will forked your PR version and change Chinese terminology on my own. Anyway, thanks for your effort!

Best,

**Comment by zhangrui-wolf** [comment]
Sorry, something has slowed down the correction.

I have fixed all the Chinese documentation based on the previous feedback. On my computer, the English documentation and the Chinese documentation show the same structure, except for the installation part in get_start.md. I think I organized this section in a much clearer structure. I also refer to the "English-Chinese terminology comparison" to correct the presentation of the Chinese documentation.

There was a problem with the serial number in the "Add new backbones" section of the English document "Customize Model". The document is displayed on the local computer without problems. But in "https://mmsegmentation.readthedocs.io/en/latest/tutorials/customize_models.html" there has a problem.

Do I need to pull a new request?

Translated with www.DeepL.com/Translator (free version)

**Review by MengzhangLI — COMMENTED** [review]
> Sorry, something has slowed down the correction.
> 
> I have fixed all the Chinese documentation based on the previous feedback. On my computer, the English documentation and the Chinese documentation show the same structure, except for the installation part in get_start.md. I think I organized this section in a much clearer structure. I also refer to the "English-Chinese terminology comparison" to correct the presentation of the Chinese documentation.
> 
> There was a problem with the serial number in the "Add new backbones" section of the English document "Customize Model". The document is displayed on the local computer without problems. But in "https://mmsegmentation.readthedocs.io/en/latest/tutorials/customize_models.html" there has a problem.
> 
> Do I need to pull a new request?
> 
> Translated with [www.DeepL.com/Translator](http://www.DeepL.com/Translator) (free version)

Hi, I will check out your correction tomorrow and the finished this pr.
As for serial number in html file, you can pull a new request later.
Thanks again for your great effort!

**Comment by MengzhangLI** [comment]
Hi, @zhangrui-wolf :
Sorry for late reply, I modified some full points in your branch and git pushed it.

image

Please check it out and we planned to merge this pr as soon as possible!

Best,

**Review by MengzhangLI — APPROVED** [review]

**Pull request merged**